ej2-react/appearance/icons.md

```diff
@@ -8,7 +8,7 @@ documentation: ug
 domainurl: ##DomainURL##
 ---
 
-# React Icons Library
+# React Icons Library Appearance component
```

Comment:
Not following our syntax and need to get content review.

Reply:
It's not a component, only Icons set for all built in themes. so, need to correct this topic.